views/emails/confirm.blade.php

```diff
@@ -4,3 +4,5 @@
 
 If this was you, simply visit the following link and your account will be activated:
 {{ $url }}
+
+If you did not sign up, just simply ignore this email.
```

Comment:
Will do.
On Apr 18, 2015 10:50 AM, Franz Liedke <notifications@github.com> wrote:In views/emails/confirm.blade.php:
> @@ -4,3 +4,5 @@
>  
>  If this was you, simply visit the following link and your account will be activated:
>  {{ $url }}
> +
> +If you did not sign up, just simply ignore this email.

Can you change it to "If you did not sign up, please ignore this email."?

Also, please add a trailing newline.

—Reply to this email directly or view it on GitHub.